js/components/conceptpicker/ConceptPicker.js

```diff
@@ -83,8 +83,8 @@ define(['jquery','knockout', 'text!./ConceptPickerTemplate.html', './InputTypes/
 			self.importValues("");			
 			return; // Nothing to do, import values is blank.
 		}
-		
-		var importConceptIds = this.importValues().split(',').map(function (e) { return +e; });
+		var parsedItems = JSON.parse(this.importValues());
+		var importConceptIds = parsedItems.items.map(item => new ConceptSetItem(item).concept.CONCEPT_ID);
```

Comment:
How this reflects "isExcluded", "includeDescendants" and "includeMapped" from the given JSON?

Reply:
The original implementation of this import function was to import by concept_id. you can see that the prior code split a string on `,`, then de-duplicated the list (on old line 89), and it would use the vocabulary provider to resolve the conceptID to  the concept object.

If you want to completely change the import implementation to accept a JSON string of an expression instead of a conceptId, that's fine (tho not backwards compatable), but you will need to look further into the code to recognize that if you are importing a json expression of a concept set, then you're not going to use the vocabulary provider to resolve the concepts (they will already appear in the JSON).

Maybe you want to split this function into 2 modes: import by concept_id and import json expression?

Reply:
@chrisknoll Split import functionality to two modes:
1) Import by Ids
2) Import JSON